lib/sycamore/sycamore/tests/unit/data/test_bbox.py

```diff
@@ -27,3 +30,11 @@ def test_union():
 def test_absolute_relative():
     assert bbox1.area == bbox1.to_relative(100, 100).to_absolute(100, 100).area
     assert isclose(bbox1.to_relative(100, 100).area, 0.16)
+
+
+def test_empty_bbox():
+    assert bbox5.is_empty()
+    assert bbox4.is_empty()
+    assert bbox4.area == 0 and bbox5.area == 0
+    assert bbox4.union_self(bbox5).area == 0 and bbox4.union_self(bbox5).is_empty()
```

Comment:
Do you want union here instead of union_self? I guess it doesn't matter since in each case the boxes are empty, but seems a bit weird to update bbox4 in place, and is counter to what you did with intersect below.

Reply:
Makes sense, have updated it